scripts/hail_batch/hgdp1kg_tobwgs_pca/hgdp_1kg_tob_wgs_pca.py

```diff
@@ -0,0 +1,98 @@
+"""Generates loadings, scores, and eigenvalues for the HGDP,1KG, and tob-wgs dataset"""
+
+import click
+import pandas as pd
+import hail as hl
+from hail.experimental import lgt_to_gt
+
+GNOMAD_HGDP_1KG_MT = (
+    'gs://gcp-public-data--gnomad/release/3.1/mt/genomes/'
+    'gnomad.genomes.v3.1.hgdp_1kg_subset_dense.mt'
+)
+
+TOB_WGS = 'gs://cpg-tob-wgs-temporary/joint_vcf/v1/work/combiner/genomes.mt'
```

Comment:
We shouldn't add an input dependency on the `temporary` bucket. Let's work with @vladsaveliev to make that available in `main`.

Reply:
sounds good! I'll add in the `main` version once it's ready

Reply:
Moved it to main :) 
`gs://cpg-tob-wgs-main/joint_vcf/v1/raw/genomes.mt/`

Reply:
thanks Vlad!